src/modules/map/components/openlayers/OpenLayersMouseTracker.vue

```diff
@@ -40,6 +40,9 @@ function showCoordinateLabel(displayedFormat) {
     return displayedFormat?.id === LV95Format.id || displayedFormat?.id === LV03Format.id
 }
 function setDisplayedFormatWithId() {
+    store.dispatch('setDisplayedFormatId', {
```

Comment:
@ltshb
I have decided to use the mouse tracker selection instead of the projection button to access different coordinate formats(though lat-lon might need some visual improvements), is that good like that?

Reply:
Good idea 👍🏼